doc/source/whatsnew/v0.19.1.txt

```diff
@@ -30,3 +30,5 @@ Performance Improvements
 
 Bug Fixes
 ~~~~~~~~~
+
+- Bug in multiple offset aliases with decimal points regarded as ints (e.g. 0.5s as 5s) (:issue:`8419`)
```

Comment:
Can you move this to v0.20.0 file and add it to the enhancements section? It was a bug that is was parsed incorrectly and did not error, but supporting fractional resolutions is an enhancement that can be listed in the enhancement section